libs/utils/env.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, target_conf=None, test_conf=None, wipe=True,
         self.ftrace = None
         self.workdir = WORKING_DIR_DEFAULT
         self.__tools = []
+        self.__installed_tools = set()
```

Comment:
We need to push a clean-up patch which removes these horrible double underscores... will do after merging this patch ;-)